src/main/java/salesforce/ui/PageTransporter.java

```diff
@@ -0,0 +1,20 @@
+package salesforce.ui;
+
+import core.WebDriverManagerSingleton;
+import salesforce.ui.pages.ProductPage;
+import utils.SetUp;
+
+/**
+ * Controls navigation of pages.
+ */
+public class PageTransporter {
+
+    public void goToUrl(final String url) {
```

Comment:
Java doc is missing is some methods of this class

Reply:
Thanks, I have added that